src/actions/guides/database/orms_and_avram.cr

```diff
@@ -0,0 +1,29 @@
+class Guides::Database::ORMsAndAvram < GuideAction
+  guide_route "/database/orms-and-avram"
+
+  def self.title
+    "ORMs and Avram"
+  end
+
+  def markdown
+    <<-MD
+    ## Avram
+
+    A note about the name "Avram". Link to the Avram repo. Lucky decision to focus on Postgres.
+
+    ## Parts to Avram
+
+    * Models
+    * Queries
+    * Forms
+    * Boxes
+    * Migrations
```

Comment:
I love this 👍 Maybe they are each their own headline though so they get shown in the table of contents